### ISSUE TITLE
List files in zip only from currentFolder folder

### ISSUE THREAD

**Issue by aldas** [Chumper/Zipper#78]
Currently `listFiles()` returns array of all files within zip. It would be handy to have method that takes `currentFolder` into consideration.

I tried to use `$files = $zipper->folder('sql')->listFiles();`  but realized after looking into code that it is not possible.

**Comment by aldas** [comment]
I could make PR for it but if it is a correct thing to do. It caused side effects for existing code.

as there could be code with
```
...
$zipper->folder('sql');
... 
doSomethingInteresting();
...
$allFilesInZip = $zipper->listFiles();
doSomethingWithAllFiles();
....
```

**Comment by aldas** [comment]
resolved with #82 now I can do same with `$zipper->listFiles('/^sql\//i);`
